### ISSUE TITLE
asyncio redis problem

### ISSUE THREAD

**Issue by afeena** [mushorg/tanner#259]
I got this exception on the server. Seems like a problem with asyncio_redis

```
2018-06-29 09:07:50 ERROR:asyncio:run_app: Task exception was never retrieved
future: <Task finished coro=<CommandCreator._get_wrapped_method.<locals>.wrapper.<locals>.done() done, defined at /usr/lib/python3.6/site-packages/asyncio_redis/protocol.py:650> exception=InvalidStateError('invalid state',)>
Traceback (most recent call last):
  File "/usr/lib/python3.6/site-packages/asyncio_redis/protocol.py", line 655, in done
    future2.set_result(result)
asyncio.base_futures.InvalidStateError: invalid state
```

**Comment by afeena** [comment]
I suggest to migrate to https://github.com/aio-libs/aioredis

**Comment by afeena** [comment]
Fixed in v0.6
